README.md

```diff
@@ -1,46 +1,87 @@
-![Passage Swift](https://storage.googleapis.com/passage-docs/passage-github-banner.png)
+![passage-flex-ios](https://storage.googleapis.com/passage-docs/github-md-assets/passage-flex-ios.png)
 
-### Native passkey authentication for your iOS app
-## Welcome!
-Integrating passkey technology can be really hard. That's why we built the PassageFlex iOS SDK - to make passkey authentication easy for you and your users.
-<br>
+![SPM Version](https://img.shields.io/github/v/release/passageidentity/passage-flex-ios?style=flat&label=Swift%20Package)
+![Cocoapods Version](https://img.shields.io/github/v/release/passageidentity/passage-flex-ios?style=flat&label=CocoaPods)
+![Language](https://img.shields.io/badge/Swift-informational?style=flat&logo=swift&logoColor=white&color=FA7343)
+![License](https://img.shields.io/github/license/passageidentity/passage-flex-ios.svg?style=flat) ![Static Badge](https://img.shields.io/badge/Built_by_1Password-grey?logo=1password)
 
-## Usage
+## About
 
-### Installation
+[Passage by 1Password](https://1password.com/product/passage) unlocks the passwordless future with a simpler, more secure passkey authentication experience. Passage handles the complexities of the [WebAuthn API](https://blog.1password.com/what-is-webauthn/), and allows you to implement passkeys with ease.
+
+Use [Passkey Flex](https://docs.passage.id/flex) to add passkeys to an existing authentication experience.
+
+Use [Passkey Complete](https://docs.passage.id/complete) as a standalone passwordless auth solution.
+
+Use [Passkey Ready](https://docs.passage.id/passkey-ready) to determine if your users are ready for passkeys.
+
+### In passage-flex-ios
+
+Use passage-swift to implement Passkey Flex in your Swift application to add native passkey authentication in your own authentication flows.
```

Comment:
I think this is wrong. we use passage-flex-ios in only iOS Applications, and passage-swift for all Apple platforms. @rickycpadilla, can you please confirm this?

Reply:
@SinaSeylani are we waiting for confirmation for these?

Reply:
I've updated the repo description to reflect this, do these tags need to be updated/any removed? I imagine macOS/tvos/watchos might need to go?

swift authentication passkeys passwordless webauthn apple ios macos passkey-flex tvos watchos biometrics 1password passage passage-sdk

Reply:
I believe passage-flex-ios _does_ support MacOS and tvOS:
https://github.com/passageidentity/passage-flex-ios/blob/main/Package.swift#L8

This was one of those naming things we haven't changed yet (though we could) to better reflect the support.

@rickycpadilla can you confirm my assumptions about what passage-flex-ios supports?

Reply:
@jenn-macfarlane  @agebhardt  yes that's correct
passage-flex-ios ios, macos, tvos
passage-swift supports All the Apple platforms

Reply:
Ok, so what's the recommendation here? I need to confirm what we want for the following:

**Repo description:** 

`Passkey Flex for iOS - Add native passkey authentication to your own Swift authentication flows with Passage by 1Password`

**Tags/keywords:**

- swift
- authentication
- passkeys
- passwordless
- webauthn
- apple
- ios
- macos
- passkey-flex
- tvos
- watchos
- biometrics
- 1password
- passage
- passage-sdk

Reply:
I'd like to recommend we move forward with the original guidance here, @jenn-macfarlane -- while "Apple platforms" is slightly more generic, I think it reads better and offers some simplicity. I don't anticipate problems for folks.

"ios, macos, and tvos" are all apple platforms
while there are _more_ apple platforms (like what passage-swift supports), I'd like to nudge us to use similar language.

Reply:
Maybe a bit pedantic but "ios, macos, and tvos" does not include watchOS so I wanted to confirm.

I'm good to go with the OG guidance, meaning:

`Passkey Complete for Apple Platforms - Go completely passwordless with a standalone auth solution in your Swift apps with Passage by 1Password`

and the list of keywords above unchanged.

Reply:
How about:

`Passkey Flex for Apple Platforms - Go completely passwordless with a standalone auth solution in your iOS, MacOS, and tvOS apps with Passage by 1Password`

Reply:
Let's remove `watchos` from the list of keywords

Reply:
Done!